### ISSUE TITLE
Could not find `Build` in `gcc` (caused by renaming of gcc-rs to cc-rs?)

### ISSUE THREAD

**Issue by Ruin0x11** [ihalila/pdcurses-sys#3]
Trying to install the latest pancurses on Windows 8 causes this error related to version `0.5.1` of `pdcurses-sys`:
```
error[E0433]: failed to resolve. Could not find `Build` in `gcc`
 --> C:\Users\ruin\.cargo\registry\src\github.com-1ecc6299db9ec823\pdcurses-sys-0.5.1\build.rs:8:10
  |
8 |     gcc::Build::new()
  |          ^^^^^ Could not find `Build` in `gcc`
```

This might have been caused by `gcc-rs` being renamed to `cc-rs`.

**Comment by ihalila** [comment]
Hmm, odd. Building 0.5.1 still worked for me. I released 0.6 that uses the latest version of cc-rs, see if that fixes the issue for you.

**Comment by Ruin0x11** [comment]
Yes, it does. Thank you.
